### PR TITLE
Remove Data-JSON and generating an unique session id

### PR DESCRIPTION
**Pull request by duboisp**
As discussed during PR review meeting, it was decided to move the Data-JSON into the GCweb theme.

The unique session id is to support a feature used in data-json and data-ajax

Related to PR #7875

### CODE REVIEW

File: src/core/wb.js [hunk 1]
```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```
Comment (LaurentGoderre) [inline src/core/wb.js]: Why is this being added?

Comment (duboisp) [inline src/core/wb.js]: For now it support cache busting in data-ajax and in data-json

Comment (nschonni) [inline src/core/wb.js]: That's something for a separate PR. Cache busting is also better by just using the date in milliseconds

Comment (duboisp) [inline src/core/wb.js]: There is different cache busting strategy that will be available in data-ajax and data-json. The author is free to use the one that best fit his scenario. 

The cache busting for data-ajax plugin is already in a different PR, see #7983

Comment (nschonni) [inline src/core/wb.js]: Ok, the session stuff can be discussed in the other PR, but it doesn't belong with the Data-JSON removal

Comment (LaurentGoderre) [inline src/core/wb.js]: I agree with @nschonni it should be a separate pr

Comment (duboisp) [inline src/core/wb.js]: That function is required by data-json which it will be added in GCWeb by PR https://github.com/wet-boew/GCWeb/pull/1237

Comment (LaurentGoderre) [inline src/core/wb.js]: It can still be dealt with separate PRs.

Comment (duboisp) [inline src/core/wb.js]: @LaurentGoderre no. The [GCweb PR](https://github.com/wet-boew/GCWeb/pull/1237) that re-add the data-json plugin need that function. Also, that function is needed by PR #7983 which add a cache buster for data-ajax.

Let discuss at the next PR review meeting.

/cc @lucas-hay

Comment (LaurentGoderre) [inline src/core/wb.js]: If it's needed by #7983 needs this why not move it to that PR?

Comment (duboisp) [inline src/core/wb.js]: Because it's needed by https://github.com/wet-boew/GCWeb/pull/1237 which are independent from #7983

Comment (LaurentGoderre) [inline src/core/wb.js]: Then create a new PR on which both wet-boew/GCWeb#1237 and #7983 would be dependent

Comment (duboisp) [inline src/core/wb.js]: That was already done by the unapproved PR #7875

Comment (LaurentGoderre) [inline src/core/wb.js]: A PR with just this code.

Comment (duboisp) [inline src/core/wb.js]: What is the issue, exception of how it was packaged? That insertion is already in a separate commit https://github.com/wet-boew/wet-boew/pull/7982/commits/a42b4cd3ef39f345bbeb2138ddf3ca0beddbed42

Comment (LaurentGoderre) [inline src/core/wb.js]: The change is unrelated to the title of the PR

**Review by lucas-hay — APPROVED** [review]
Approved at PR review meeting.
